testing/cffi1/test_function_args.py

```diff
@@ -1,4 +1,9 @@
 import pytest, sys
+
+pytestmark = [
+    pytest.mark.thread_unsafe(reason="Workers would share a build directory"),
+]
```

Comment:
this wasn't needed until now because this test file only runs on Windows, see below